### PR TITLE
PHP 8.4: Document SplObjectStorage::seek()

### PR DESCRIPTION
**Pull request by Girgias**
@nielsdos could you document this as, IIRC, you added this?

Related (#4047)

### CODE REVIEW

**Comment by nielsdos** [comment]
Pushed commits, including 2 drive-by fixes:
1) one simpara fix because I copied that section from that file and noticed it
2) Documenting ArrayIterator

File: reference/spl/splobjectstorage/seek.xml [hunk 1]
```diff
@@ -11,26 +11,26 @@
...
      <listitem>
       <simpara>
-       Description.
+       The position to seek to.
       </simpara>
      </listitem>
     </varlistentry>
    </variablelist>
-  </simpara>
+  </para>
```
Comment (Girgias) [inline reference/spl/splobjectstorage/seek.xml]: Ideally this wrapping para should be removed, but I'm fine to leave the `<para>` tag if it is pre-exiting

File: reference/spl/splobjectstorage/seek.xml [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    <varlistentry>
+     <term><parameter>offset</parameter></term>
+     <listitem>
+      <simpara>
+       The position to seek to.
+      </simpara>
+     </listitem>
+    </varlistentry>
+   </variablelist>
+  </para>
```
Comment (Girgias) [inline reference/spl/splobjectstorage/seek.xml:33]:
```suggestion
  <variablelist>
   <varlistentry>
    <term><parameter>offset</parameter></term>
    <listitem>
     <simpara>
      The position to seek to.
     </simpara>
    </listitem>
   </varlistentry>
  </variablelist>
```

File: reference/spl/splobjectstorage/seek.xml (hunk 1 above)
Comment (nielsdos) [reply reference/spl/splobjectstorage/seek.xml]: The simpara came out of the stub generator, but that was wrong so I changed it back to para: https://github.com/php/php-src/pull/16764

**Comment by nielsdos** [comment]
I think this could be merged, no?

**Pull request merged**